utils/api/updateBlog.ts

```diff
@@ -0,0 +1,12 @@
+import type { BlogIdRequestBody } from '../../pages/api/blogs/[blogId]';
+
+export const updateBlog = (blogId: string, body: BlogIdRequestBody) => {
+  body.slug = body.slug || null;
+  body.favicon = body.favicon || null;
+  body.creatorEmail = body.creatorEmail || null;
+
```

Comment:
Myślę, że lepiej tę weryfikację robić jednak po stronie serwera, bo to nam chodzi o to by nie było pustych stringów w bazie danych, dla usera nie ma to znaczenia i nie widzi on żadnej różnicy